requirements/runtime.txt

```diff
@@ -4,5 +4,6 @@ numpy
 pyyaml
 regex;sys_platform=='win32'
 rich
+tabulate
```

Comment:
Can we avoid adding the new dependency `tabulate` by using [the table of rich](https://rich.readthedocs.io/en/latest/tables.html)?

Reply:
Okk, I'll do it.

Reply:
The code has been updated, please review.